packages/asc-ui/src/components/SearchBar/SearchBar.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
+import { ReactComponent as Search } from '@datapunt/asc-assets/lib/Icons/Search.svg'
+import styled from '../../styled-components'
+import IconButton from '../IconButton'
+import { InputStyle } from '../../styles/components'
+import IconButtonStyle from '../../styles/components/IconButtonStyle'
+import SearchBarStyle from '../../styles/components/SearchBarStyle'
+import { KeyboardKeys } from '../../types'
+import Input from '../Input'
+
+type SearchBarProps = {
+  minWidth?: string
+  maxWidth?: string
+  padding?: string
+  styledComponent?: any
```

Comment:
No any plz, unless it really does not matter ;) Look at the '../../styled-components' file or inspect styled-components files to get the proper type

Reply:
Do you know which type? @chrisvanmook